src/stan/model/indexing/lvalue.hpp

```diff
@@ -118,7 +118,7 @@ inline void assign(Eigen::Matrix<T, 1, Eigen::Dynamic>& x,
  * the indexed size.
  */
 template <typename T, typename I, typename U>
-inline typename std::enable_if<!std::is_same<I, index_uni>::value, void>::type
+inline typename std::enable_if_t<!std::is_same<I, index_uni>::value>::type
```

Comment:
need to get rid of the typename and ::type

Reply:
Yes, I am a fool :) Thanks!